### PR TITLE
Lesson16 fercan

### PR DESCRIPTION
**Pull request by fers9021**
first pull request to lesson16

### CODE REVIEW

**Comment by fers9021** [comment]
did ex 12 and 14

File: Lesson16/lesson 16-fercan/script.js [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+    console.log('It\'s a positive number');
+ } else { 
+    console.log ('It\'s equal to zero.');
+}}
+
+
+// 6. Write a function that takes in a year and checks if it's a leap year 
+//(is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
+
+function isLeapYear (year) {
```
Comment (annasvst) [inline Lesson16/lesson 16-fercan/script.js:31]: In if...else statement you can use 3 parts:
1. `if (condition)` - initial statement with condition
2. `else if (condition)` - additional condition (cannot be the first statement!!!)
3. `else` - can only be the last statement, cannot have a condition.

So you function should look like this instead:
```
function isLeapYear (year) {

  if(year%4==0 && year%100!=0){
    console.log('It\'s a leap year');
  } else if (year%400==0); {  
    console.log('It\'s a leap year');
  } else {
    console.log('It\'s not a leap year');
  }
}
```

File: Lesson16/lesson 16-fercan/script.js [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+getPersonsAge(86);
+getPersonsAge(12);
+
+// 10. Write a function that takes age as a number and whether they are a student (yes or no). 
+//If the age is less than 18 and the user is a student, print "You are a student and not old enough to vote". 
+//If the age is between 18 and 65 (inclusive) and the user is not a student, print "You are eligible to vote". 
+//If the age is greater than 65, print "You are a senior citizen". 
+//If the user is a student but their age is greater than or equal to 18, print "You are old enough to vote".
+
+function getOnePersonsAgeAndStatus (num, status) {
```
Comment (annasvst) [inline Lesson16/lesson 16-fercan/script.js:130]: In cases like this where status can only take 2 states (is student or not a student), it's common in programming to use a boolean type variable. The function would look like this instead:
```
function getOnePersonsAgeAndStatus (num, isStudent) {
  if (num<18 && isStudent) {
    console.log('You are a student and not old enough to vote');
  }
  else if (num>=18 && num<=65 && !isStudent) {
    console.log('You are eligible to vote');
  }
  else if (num>65) {
    console.log('You are a senior citizen');
  }
  else (num>=18 && isStudent) 
    console.log('You are old enough to vote');
  }


getOnePersonsAgeAndStatus(36, true);
getOnePersonsAgeAndStatus(68, false);

```

You solution works so don't change it now but try using boolean type instead in the future :)

**Review by annasvst — APPROVED** [review]
Approved

**Pull request merged**